src/twisted/conch/ssh/keys.py

```diff
@@ -955,9 +1081,16 @@ def blob(self):
             integer g
             integer y
 
+        EC keys::
+            string 'ecdsa-sha2-[identifier]'
+            integer x
+            integer y
+
+            identifier is the standard NIST curve name
+
         @rtype: L{bytes}
         """
-        type = self.type()
+        type = self.type() #takes care of bad key type.
```

Comment:
I realize that you added these comments as a result of @Lukasa's feedback, but this isn't a great way to address that concern.  We don't put comments on every function call everywhere indicating what exceptions might be raised :).  Instead, documenting this on `type` itself, and replying to @Lukasa's comments in-line, would be better.